src/components/molecules/input/ms-input.stories.ts

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from '@storybook/web-components'
+import { html } from 'lit'
+
+import './ms-input'
+
+export default {
+  title: 'Input',
+  argTypes: {
+    value: { defaultValue: '' },
+    placeholder: { defaultValue: '' },
+    disabled: { options: [true, false], control: { type: 'select' } },
+    label: { defaultValue: 'Test' },
+    message: { defaultValue: 'Informative message' },
+    status: { options: ['error', 'success', ''], control: { type: 'select' } },
+  },
+  render: (args) =>
+    html`<ms-input
+      .value=${args.value}
+      ?disabled=${args.disabled}
+      .status=${args.status}
+      placeholder=${args.placeholder}
```

Comment:
I believe the dot is needed before the `placeholder` prop

Reply:
done!